lib/general.sh

```diff
@@ -671,7 +676,6 @@ prepare_host()
 		"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-5.5.0-2017.10-x86_64_arm-linux-gnueabihf.tar.xz"
 		"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-6.4.1-2017.11-x86_64_arm-linux-gnueabihf.tar.xz"
 		"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-6.4.1-2017.11-x86_64_aarch64-linux-gnu.tar.xz"
-		"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-7.4.1-2019.02-x86_64_aarch64-linux-gnu.tar.xz"
```

Comment:
Why is this getting removed?

Reply:
duplicate line

```
"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-6.4.1-2017.11-x86_64_aarch64-linux-gnu.tar.xz"
"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-7.4.1-2019.02-x86_64_aarch64-linux-gnu.tar.xz"
"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-7.4.1-2019.02-x86_64_arm-linux-gnueabihf.tar.xz"
"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-7.4.1-2019.02-x86_64_arm-eabi.tar.xz"
"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-7.4.1-2019.02-x86_64_arm-linux-gnueabi.tar.xz"
"https://${ARMBIANSERVER}/_toolchains/gcc-linaro-7.4.1-2019.02-x86_64_aarch64-linux-gnu.tar.xz"
```

Reply:
Sometimes we need older version of certain compiler to be able to compile certain sources ... or waste time on adjusting those sources. It is more convenient to have this older compiler by hand. To be sure if we can remove this compiler, we need to check those sources that needs it ... it more recent is also good. Leave it inside for now.

Reply:
> Sometimes we need older version of certain compiler to be able to compile certain sources ... or waste time on adjusting those sources. It is more convenient to have this older compiler by hand. To be sure if we can remove this compiler, we need to check those sources that needs it ... it more recent is also good. Leave it inside for now.

`gcc-linaro-7.4.1-2019.02-x86_64_aarch64-linux-gnu.tar.xz`
this package is duplicated，so I remove the duplicated link, please watch carefully

Reply:
Oh, I see. True ... and I did check twice :) Sorry for the noise.